src/tidecli/models/task_data.py

```diff
@@ -147,4 +148,12 @@ class TideCourseData(BaseModel):
     Model for Tide course data.
 
     """
-    weeks: Dict[str, TideWeekData] = {}
+    course_parts: Dict[str, TideCoursePartData] = {}
+
+
+class RootFolders:
```

Comment:
I'd prefer singular class name here, not a strong opinion though.

Reply:
I would reserve course_part for single part and collection of them is many